app/assets/stylesheets/_authors.scss

```diff
@@ -1,9 +1,39 @@
 .author__name {
-  font-size: 1.5em;
+  font-size: $base-title-size;
 }
 
 .author__books {
-  @include display(flex);
-  @include border-color(null null #CCCCCC null);
+  @include border-color(null null $base-secondary-color null);
   @include border-style(null null solid null);
+  @include display(flex);
+}
+
+.author-cover {
+  @include span-columns(4);
 }
+
+.author-info {
+  @include span-columns(8);
+  @include omega();
+  span {
+    font-weight:bold;
+  }
+}
+.author-page {
+  .books{
+    clear:left;
+    @include shift(4);
```

Comment:
Expected item on line 25 to appear before line 24. Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets